test/JsBuiltIns/rlexe.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<regress-exe>
+  <test>
+    <default>
+      <files>ChakraLib.js</files>
+      <compile-flags>-args summary -endargs -LdChakraLib</compile-flags>
```

Comment:
The LdChakraLib is used to enable/disable the __chakraLibrary global variable used to access to the internal C++ methods from Chakra like toLength and toNumber. It must be used only when we generate the bytecode of the JsBuiltIn.js file.
The JsBuiltIn flag is used to enable/disable the JsBuiltInEngineExtension in C++. When this flag is disabled, the previous version of the built ins functions (the C++ version of IndexOf) take the relay.
Those two flags are independent but will mainly be used together. A use case of disabling JsBuiltIn would be to do some debugging stuff or compare performances without having to retrieve an exe or dll from chakrafs.

---
In reply to: [115902305](https://github.com/Microsoft/ChakraCore/pull/2898#discussion_r115902305) [](ancestors = 115902305)